README.md

```diff
@@ -119,6 +119,15 @@ Lint all `yaml` files in `path` and all subdirectories (recursive):
 - 12 is an warning and informational
 - 14 is an error and a warning and an informational
 
+###### Configuring Exit Codes
+`cfn-lint` will allow you to configure exit codes. You can provided the parameter `--non-zero-exit-code` with a value of `informational`, `warning`, `error`, or `none`. `cfn-lint` will determine the exit code based on the match severity being the value of the parameter `--non-zero-exit-code` and higher. The exit codes will remain the same as above.
```

Comment:
`cfn-lint` allows you to configure exit codes. You can provide the parameter `--non-zero-exit-code` with a value of

Reply:
Small grammar suggestion here. Change will allow to allow and provided to provide.